### PR TITLE
🚧 BUU: some design updates + one small code fix

### PR DESCRIPTION
**Pull request by jibees**
#### What? Why?

Follow-up https://github.com/openfoodfoundation/openfoodnetwork/pull/11007

Separate WIP in this branch to several PRs.

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Review by rioug — APPROVED** [review]
Looks good !

File: app/webpacker/css/admin/products_v3.scss [hunk 1]
```diff
@@ -115,8 +114,12 @@
...
 
   #filters {
-    gap: 20px;
-    align-items: flex-end;
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: 1fr;
+    grid-column-gap: 20px;
+    align-items: end;
+    margin-bottom: 20px;
```
Comment (dacook) [inline app/webpacker/css/admin/products_v3.scss:122]: I think we might have had this conversation before, but I can't remember:

I find the grid syntax not as clear for what it is doing. I think I prefer the old style of using classes like `.two.columns` , but admit that is limited. This is probably good, I just need to get more familiar with it :)

File: app/webpacker/css/admin_v3/components/tom_select.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+.ts-wrapper.multi {
+  min-height: 40px;
+
+  &.input-active {
+    .ts-control {
+      border-color: $lighter-grey;
+
+      input {
+        &::placeholder {
+          color: $light-grey;
```
Comment (dacook) [inline app/webpacker/css/admin_v3/components/tom_select.scss:11]: I noticed this was barely readable

File: app/views/admin/products_v3/_filters.html.haml [hunk 3]
```diff
@@ -4,10 +4,10 @@
       = text_field_tag :search_term, search_term, placeholder: t('.search_products')
   .producers
     .label= t('.producers.label')
-    = select_tag :producer_id, options_for_select(producer_options, producer_id), include_blank: t('.all_producers')
+    = select_tag :producer_id, options_for_select(producer_options, producer_id), include_blank: t('.all_producers'), "data-controller": "tom-select", "data-tom-select-options-value": '{ "plugins": [] }', class: "fullwidth"
```
Comment (dacook) [inline app/views/admin/products_v3/_filters.html.haml:7]: Perhaps we want to make a `tom_select_tag` helper, with predefined options for "single", etc. We can style it with a class.

**Review by dacook — APPROVED** [review]
Looks good for now 👍

**Comment by dacook** [comment]
This is behind a WIP feature toggle, and doesn't solve any outcomes that need testing, so I'll merge now.

**Pull request merged**